benchmarks/benchmark-local-circuit-random.eval.ts

```diff
@@ -35,7 +35,7 @@ evalite.experimental_skip("Electronics Engineer Making Random Circuits", {
       systemPrompt,
       logsDir,
       maxAttempts: 4,
-      promptNumber: ++promptNumber,
+      promptId: ++promptNumber,
```

Comment:
fwiw i like to prefix ids with `prompt_${number}` so that it's easier to track ids and make sure they don't become polymorphic